static/js/util/profile_edit.js

```diff
@@ -281,44 +281,10 @@ export function boundCountrySelectField(stateKeySet, countryKeySet, label) {
  *
  * @param keySet {String[]} Path to look up and set a field
  * @param label {String} Label for the field
+ * @param omitDate {bool} If false, there is a date textbox in addition to month and year
```

Comment:
moment.js uses `date` to refer to the day of month and `day` to refer to day of month. But most other libraries refer to day like you said so I'll change the references